app/templates/join.hbs

```diff
@@ -1,11 +1,24 @@
 {{page-title "Join"}}
 
 <div class="container mx-auto lg:max-w-screen-lg py-16 pb-48 px-3 md:px-6">
-  <h1 class="text-2xl font-bold text-gray-700 md:text-3xl text-center">
-    Join the best.
-  </h1>
+  <AffiliateLinkPage::HeroSection />
+  <AffiliateLinkPage::AcceptReferralContainer @affiliateLink={{this.model.affiliateLink}} class="mt-20 md:mt-24 mb-52" />
 
-  <AffiliateLinkPage::AcceptReferralContainer @affiliateLink={{this.model.affiliateLink}} class="mt-8" />
+  <div class="space-y-6 text-center my-8 max-w-[640px] max-h-[198px] mx-auto">
+    <h2 class="text-3xl md:text-4xl font-bold text-gray-900">Trusted by senior engineers at
+      <span class="italic font-extrabold">the world's best companies</span></h2>
+    <p class="text-sm md:text-lg text-gray-800">Discover how core contributors to Docker and Next.js practice writing code at the edge of their
+      abilities — as well as senior engineers at companies like Apple, NVIDIA, and Stripe.</p>
+  </div>
+
+  <div><h2 class="text-gray-400 uppercase text-center text-xs mb-5">Trusted by engineers at top companies</h2>
+    <AffiliateLinkPage::LogoCloud />
+  </div>
+
+  <img src={{this.ccBanner}} alt="CC Banner" class="w-4/5 mx-auto mt-16" />
+
+  <AffiliateLinkPage::AcceptReferralButton @affiliateLink={{this.model.affiliateLink}} class="mt-16" />
```

Comment:
When you have "grouped" elements like this, it's often more readable if you use `gap-4` instead of `mt-4` on the second element. i.e.: 

```
<div class="flex flex-col items-center gap-y-4">
  <Button>
  <Text>
</div>
```

Also, the only reason the button is centered at the moment is because there's the `flex items-center justify-center` stuff inside that component - layout/positioning is cleaner when its the responsibility of the caller. If using a lower-level component (button, toggle etc) automatically forces a specific layout, that's often hard to deal with when the usage context changes.

Reply:
@rohitpaulk Thank you, all these comments are very helpful ❤️